CHANGES.rst

```diff
@@ -34,6 +35,12 @@ outlier_detection
 - Removed ``grow`` from the ``outlier_detection`` step parameters,
   because it's no longer used in the algorithms. [#8190]
 
+photom
+------
+
+- Updated photom step to include spectral dispersion in NIRCAM WFSS and TSGRISM
```

Comment:
It would be good to be a bit more explicit here and say something like "... to account for the dispersion per pixel when applying the flux calibration data to NIRCam WFSS and TSGRISM images."

[Some of us - not naming any names - are responsible for writing up user-friendly release notes for each build release, and sometimes it's really hard to figure out exactly what changed, how it affects the data, and why users should care when coming back to these cryptic change log entries 3 months later.]